### ISSUE TITLE
Become part of the OpenSSF Best Practices Badge Program

### ISSUE THREAD

**Issue by OleMussmann** [yatiml/yatiml#46]
During the review of the package "howfairis" https://github.com/fair-software/howfairis we saw that yatiml is not part of the OpenSSF Best Practices Badge Program yet. The README.md also misses the fair-software.eu badge. Output from `howfairis https://github.com/yatiml/yatiml`:

```
Checking compliance with fair-software.eu...
url: https://github.com/yatiml/yatiml
(1/5) repository
      ✓ has_open_repository
(2/5) license
      ✓ has_license
(3/5) registry
      × has_ascl_badge
      × has_bintray_badge
      × has_conda_badge
      × has_cran_badge
      × has_crates_badge
      × has_maven_badge
      × has_npm_badge
      × has_pypi_badge
      ✓ has_rsd_badge
      × is_on_github_marketplace
(4/5) citation
      × has_citation_file
      ✓ has_citationcff_file
      × has_codemeta_file
      ✓ has_zenodo_badge
      × has_zenodo_metadata_file
(5/5) checklist
      × has_core_infrastructures_badge

Calculated compliance: ● ● ● ● ○

It seems you have not yet added the fair-software.eu badge to your README.rst. You can do so by pasting the following snippet:

.. image:: https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8B-yellow
   :target: https://fair-software.eu
```

**Comment by LourensVeen** [comment]
I can add the fair-software.eu badge, it's not much effort.

As for the core infrastructure thing, I started that on one of my repos and it's a lot of work. If I had a hundred users, major packages depending on yatiml, regular external contributors and a budget to maintain this thing then I'd do it, but as long as YAtiML is a small Open Source project that I work on in my spare time, new features and bug fixes rank higher on the list of priorities and this will be a wontfix.

**Comment by LourensVeen** [comment]
I've added the badge, and I'm going to pass on the OpenSSF thing, so this one is ready to go.

**Comment by LourensVeen** [comment]
Released in 0.10.0.
